src/components/popper/PopperComponent.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+import { Manager, Reference, Popper } from 'react-popper';
+import type { Modifiers, Placement } from 'popper.js';
+import { PLACEMENT_AUTO } from './constants';
+
+type Props = {
+    children: React.Node,
+    isOpen?: boolean,
+    modifiers?: Modifiers,
+    placement?: Placement,
+};
+
+const PopperComponent = (props: Props) => {
+    const { children, isOpen, modifiers, placement: popperPlacement = PLACEMENT_AUTO } = props;
```

Comment:
Move the placement defaulting to PopperComponent.defaultProps below and remove the ? From it's type